examples/pipeline.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: test-pipeline
+spec:
+  description: >-
+    This is an integration pipeline example
+  params:
+    - description: 'Snapshot of the application'
+      name: SNAPSHOT
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - description: 'Namespace where the application is running'
+      name: NAMESPACE
+      default: "default"
+      type: string
+    - name: test-task
+      taskRef:
```

Comment:
How does this work? Setting taskRef withou actual reference to task? I guess you ment to use taskSpec here?

Reply:
Thanks. The "test-task" is in task.yaml now.